<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
background-color:#E3EAF4;
}
QLineEdit{
background-color:#E3EAF4;
}
QDialog{
background-color:#E3EAF4;
margin-top:10px;
}
QLabel{
font-size:14px;
font-family:Microsoft YaHei;
background-color:#E3EAF4;
}
QGroupBox{
font-size:14px;
font-family:Microsoft YaHei;
border:1px solid rgb(0, 0, 0);
background-color:#E3EAF4;
}
QPushButton{
background-color:#2884D8;
color: #FFFFFF;
font-size:14px;
font-family:Microsoft YaHei;
}
QTextBrowser{
border-width:0;
border-style:outset;
background-color:#E3EAF4;
}
QGraphicsView{
background-color:#E3EAF4;
border-width:0;
border-style:outset;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,0,1,0">
      <item row="3" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_next">
          <property name="text">
           <string>下一步</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>信号源</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_freq_sg"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>功率</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>频率</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_power_sg"/>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox_sg_freq">
             <item>
              <property name="text">
               <string>MHz</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBox_sg_power">
             <item>
              <property name="text">
               <string>dBm</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>频谱仪</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_freq_sa"/>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBox_sa_bw">
             <item>
              <property name="text">
               <string>MHz</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox_sa_freq">
             <item>
              <property name="text">
               <string>MHz</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>带宽</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>频率</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_bw_sa"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconQrc.qrc"/>
 </resources>
 <connections/>
</ui>
